src/sec_certs/dataset/fips_algorithm.py

```diff
@@ -87,20 +87,81 @@ def download_alg_list_htmls(output_dir: Path) -> list[Path]:
 
         return paths
 
+    @staticmethod
+    def download_algs_data(output_dir: Path, alg_links: list[str]) -> list[Path]:
+        n_pages = len(alg_links)
```

Comment:
Since this is used just once, maybe used `len(alg_links)` directly and not introduce the variable?